luna/featurevis/featurevis.py

```diff
@@ -153,27 +188,19 @@ def gradient_ascent_step(img, model, regularization, optimization_parameters,
             learning_rate = optimization_parameters.learning_rate or 0.7
             img = img + learning_rate * grads
         else:
-            grads_relu_0 = rg.redirected_relu_grad(img, grads*-1)
-            grads_modified = rg.redirected_relu6_grad(img, grads_relu_0)
-            optimization_parameters.optimizer.apply_gradients(zip([grads_modified], [img]))
+            # Relu gradient overrides do not seem to work,
```

Comment:
I would suggest keeping the current version of the gradient override as by removing it, the problem still persists (at least with this gradient we are getting some minimal coloration)

Reply:
These are the results I get with/without the current gradient overrides.
**With gradient overrides. Transformations: scale + blur**
![test_scale_blur_with_gradient_overrides](https://user-images.githubusercontent.com/8738140/182085974-75b88cfe-2243-464b-91c8-85f144edf0bd.png)
**Without gradient overrides. Transformations: scale + blur**
![test_scale_blur_without_grad_override](https://user-images.githubusercontent.com/8738140/182085976-582c7512-6861-4ebc-91fc-d81502e68e79.png)
**With gradient overrides. Transformations: default**
![test_default_trans_with_gradient_overrides](https://user-images.githubusercontent.com/8738140/182086686-0383a036-3ac9-49a7-aa42-ab114169761a.png)
**Without gradient overrides. Transformations: default**
![test_default_transformations_without_gradient_overrides](https://user-images.githubusercontent.com/8738140/182087097-e9d43ecd-ffea-4a68-b997-15f998b04753.png)


I think that separating the transformations out of the the optimized input is the reason why the gradient overrides now impact the result (more?).

Reply:
So what is our conclusion? We remove them for now and add them when we have an idea how to make them work?

Reply:
Running the test file, at the moment, the result without gradient override looks like this:
![test](https://user-images.githubusercontent.com/52014627/182245458-bdd5fb77-14c1-45c0-b61b-6e57377a27df.png)
 
In my opinion, a "broken" gradient override is still a better result regarding both color and sharpness than something completely blurry but if you both would like to remove it, I am not against it.